soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -443,30 +462,30 @@ static bool CreateHint(RandomizerCheck hintedLocation, uint8_t copies, HintType
   //make hint text
   Text finalHint;
   Text prefix = ::Hint(RHT_PREFIX).GetText();
-  std::vector<uint8_t> colours = {QM_GREEN, QM_RED};
+  std::vector<std::string> colours = {QM_GREEN, QM_RED};
   if (type == HINT_TYPE_WOTH){
     Text regionText = GetHintRegion(ctx->GetItemLocation(hintedLocation)->GetParentRegionKey())->GetHint().GetText();
-    finalHint = prefix + "%r#" + regionText + "#%w" + ::Hint(RHT_WAY_OF_THE_HERO).GetText();
-    colours = {QM_LBLUE};
+    finalHint = prefix + "#" + regionText + "#" + ::Hint(RHT_WAY_OF_THE_HERO).GetText();
+    colours = {QM_GREEN};
```

Comment:
Any reason for the color changes here?

Reply:
I believe LBLUE is what n64rando uses.

Reply:
![image](https://github.com/HarbourMasters/Shipwright/assets/72659707/9779b397-6de6-4da2-9399-5ee70bfc3a4d)
![image](https://github.com/HarbourMasters/Shipwright/assets/72659707/4a08a36b-09d1-44b0-918c-13bf1ae1c618)

The `QM_LBLUE` & `QM_PINK` are the ones that should remain as they were added for parity with n64 rando.

Reply:
I just checked a video of N64 rando (from 2021 granted). It actually had the region colored blue/pink and no other colors. Not to say we can't change that just bringing that up.

Reply:
For now I'll change it to match develop-macready. We can always change it later.